aten/src/ATen/core/op_registration/op_registration_test.cpp

```diff
@@ -774,6 +774,32 @@ void autograd_kernel(Tensor a) {
   called_autograd = true;
 }
 
+TEST(OperatorRegistrationTest, whenRegisteringCPUTensorType_thenCanCallUnboxedWithCPUTensorIdDispatchKey) {
```

Comment:
@smessmer , let me know if this is the ideal way to add test? (is this the correct place to add unit tests? also if the way to test make sense.

Reply:
This is unrelated to autograd, so I'd rather add the test somewhere further up in the file where we test that we can dispatch to different backends. Somewhere close to whenRegisteringMultipleKernelsInSameOpCallAndCalling_thenCallsCorrectKernel